### ISSUE TITLE
Add documentation for the weave pattern example

### ISSUE THREAD

**Issue by neozhaoliang** [cduck/hyperbolic#30]
Hello: I am studying the example `isosceles-tiling.ipynb`, but I cannot understand it from the code alone. Can you add some text in the notebook to explain the logic behind drawing it?

**Comment by cduck** [comment]
Thanks for reaching out.  It's been a while since I've written that example.  What parts do you have questions about that I could clarify?

**Comment by neozhaoliang** [comment]
There are some magic numbers in the code, like

```python
pa0 = tiles[49].vertices[0]
pa1 = tiles[0].vertices[0]
pa2 = tiles[38].vertices[0]
pb0 = tiles[8].vertices[0]
pb1 = tiles[5].vertices[0]
pb2 = tiles[127].vertices[0]
```

I'm wondering how these came out. Indeed, an explanation of the main idean that how the weave pattern is construct would suffice. Otherwise, the code is like a black box and other people have no idea how to use it.

**Comment by cduck** [comment]
I see what you mean.  The tile patterns are generated by starting with the center polygon ("tile") then adding adjacent tiles around in a spiral.  So `tiles[49]` is the 50th tile in the spiral pattern.

I found those magic numbers by first generating the tiling, then locating the vertices I needed to compute the positions of other lines.  By a combination of counting, and trial-and-error, I located which numbers in `tiles[a].vertices[b]` returned the points I wanted.
